plio/queries.py

```diff
@@ -190,3 +201,79 @@ def get_events_query(plio_uuid: str, schema: str, show_unmasked_user_id: bool =
         INNER JOIN {schema}.plio AS plio ON plio.id = session.plio_id
         INNER JOIN public.user AS users ON session.user_id = users.id
         WHERE plio.uuid  = '{plio_uuid}'"""
+
+
+def get_user_level_metrics_query(
+    plio_uuid: str, schema: str, show_unmasked_user_id: bool = True
+):
+    """
+    Returns data for each user for the given plio. These include
+    - The user's identifier
+    - Whether the user has logged in via SSO
+    - Number of questions attempted by the user
+    - Number of questions answered correctly by the user
+    - Number of questions answered incorrectly by the user
+    - Number of questions skipped by the user
+    - Has the user attempted all the questions?
+    - Has the user answered all the questions correctly?
```

Comment:
Why is this needed. Looks like it's already there from the other columns?

Reply:
So we were not giving them the correct answer before. If they had correct answer, they can derive these values. But that would require them to take it into excel or pandas and do some joins.

Usually non tech people use these csvs from the plio dashboard, so that's why this info might make sense to be written into the file itself.

Also nudge people specifically asked for this :p